scapy/contrib/diameter.py

```diff
@@ -4791,7 +4794,7 @@ def getCmdParams(cmd, request, **fields):
         else:   # Assume type is int
             drAppId = val
     else:  # Application Id shall be taken from the params found based on cmd
-        drAppId = params[2].keys()[0]   # The first record is taken
+        drAppId = list(params[2].keys())[0]   # The first record is taken
```

Comment:
I think `drAppId = next(iter(params[2]))` should be enough. Useless list, again.